CONTRIBUTING.md

```diff
@@ -9,48 +9,43 @@ Please note that this project is released with a [Contributor Code of Conduct](c
 ## Table of Contents
 
 - [Adding to this list](#adding-to-this-list)
-- [Creating your own awesome list](#creating-your-own-awesome-list)
 - [Adding something to an awesome list](#adding-something-to-an-awesome-list)
 - [Updating your Pull Request](#updating-your-pull-request)
 
 ## Adding to this list
 
 Please ensure your pull request adheres to the following guidelines:
 
-- Search previous suggestions before making a new one, as yours may be a duplicate.
-- Make sure the list is useful before submitting. That implies it has enough content and every item has a good succinct description.
-- Make an individual pull request for each suggestion.
-- For Hyper **package** and **theme** names, please use the package name as published on npm, and as displayed in the URI of the package. Example: Title of `https://www.npmjs.com/package/hyperterm-snazzy` would be `hyperterm-snazzy`
-- For Hyper **resources**, use [title-casing](http://titlecapitalization.com) (AP style).
-- Use the following format: `[List Name](link) - Your awesome description`
-- Link additions should be added to the bottom of the relevant category.
-- New categories or improvements to the existing categorization are welcome.
+- Search previous requests for additions to awesome-hyper before making a new one - your request may be a duplicate.
```

Comment:
Should we use back-ticks for every occurrence of `awesome-hyper`? It's used on the next line so I'm wondering.